src/pages/lett-vanskelig/LettEllerVanskeligSpm.tsx

```diff
@@ -3,32 +3,30 @@ import * as React from "react";
 import AlternativGruppe from "../../components/alternativ-gruppe/alternativ-gruppe";
 
 interface LettEllerVanskeligSpmProps {
-    children?: null | React.ReactNode | React.ReactChild | React.ReactChildren
     nextPage?: () => void,
+    endreAlternativ: (arg: string) => void,
+    valgtAlternativ: string
 }
 
-interface State {
-    svar: string;
-}
+class LettEllerVanskeligSpm extends React.Component<LettEllerVanskeligSpmProps> {
 
-class LettEllerVanskeligSpm extends React.Component<LettEllerVanskeligSpmProps, State> {
+    static Id = "lett-vanskelig";
```

Comment:
Hva trengs denne til ?

Reply:
Denne id'en brukes for å unngå magic strings / hardkodede id'er i App.tsx. På denne måten kan idene endres kun ett sted uten å brekke hele applikasjonen.